src/platform/bouffalolab/BL602/NetworkCommissioningDriver.h

```diff
@@ -135,7 +135,7 @@ class BLWiFiDriver final : public WiFiDriver
     NetworkStatusChangeCallback * mpStatusChangeCallback = nullptr;
     int32_t mLastDisconnectedReason;
 
-    char mScanSSID[DeviceLayer::Internal::kMaxWiFiSSIDLength];
+    char mScanSSID[DeviceLayer::Internal::kMaxWiFiSSIDLength + 1];
```

Comment:
Please document that the +1 is to allow room for the null terminator.  Same in the other header.

Reply:
Done